### ISSUE TITLE
jspm not parsing package.json in Visual Studio 2013 created json file

### ISSUE THREAD

**Issue by Argorob** [jspm/jspm-cli#442]
I created a package.json file using Visual Studio 2013 with code that worked fine in other environments.  When executing "jspm install" I get an error that jspm cannot parse the package.json file.  I recreated the package.json file using notepad and saved as ascii then included the file in my VS 2013 directory.  jspm parsed the file perfectly.

**Comment by guybedford** [comment]
This could be an issue with a byte order mark perhaps? Any chance you can point me to the package.json?

**Comment by guybedford** [comment]
Please do post an example / further info if you can, in the mean time closing this issue.

**Comment by Argorob** [comment]
Hi Guy,
Sorry for the delay.  I uploaded the code here https://github.com/Argorob/AureliaSignalrv2/blob/master/AureliaSignalrv2/package.json  I will say that when I did a push to this repo I received a massage that during the push it changed all the LF to CRLF.  Not sure if that did anything.  I can email the original file but I do know when I use sublime to create the package.json file then in VS 13 do an "Include in Project" on package.json it works fine.  Maybe VS13 is putting extra hidden characters or something.  thnx for replying.  Regards.
